source/github_ee/README.md

```diff
@@ -0,0 +1,21 @@
+# github ee
+
+## Github Enterprise Edition
+
+This driver is catered for those who run Github Enterprise under private infrastructure.
+
+The below URL scheme illustrates how to source migration files from Github Enterprise.
+
+Github client for Go requires API and Uploads endpoint hosts in order to create an instance of Github Enterprise Client. We're making an assumption that the API and Uploads are available under `https://api.*` and `https://uploads.*` respectively. [Github Enterprise Installation Guide](https://help.github.com/en/enterprise/2.15/admin/installation/enabling-subdomain-isolation) recommends that you enable Subdomain isolation feature.
+
+`github-ee://user:personal-access-token@host/owner/repo/path?skipSSLVerify=false#ref`
+
+| URL Query  | WithInstance Config | Description |
+|------------|---------------------|-------------|
+| user | | The username of the user connecting |
+| personal-access-token | | Personal access token from your Github Enterprise instance |
+| owner | | the repo owner |
+| repo | | the name of the repository |
+| path | | path in repo to migrations |
+| ref | | (optional) can be a SHA, branch, or tag |
+| skipSSLVerify | | (optional) defaults to `false` |
```

Comment:
Added this option, since some GH EE instances _could_ be running on non-https or with unverified self-signed certificates. By default it will always verify ssl.

Reply:
Let's rename this to `verify-tls` with a default value of `true` and add docs about how it sets `tls.Config.InsecureSkipVerify`